CRM/Financeextras/BAO/CreditNoteAllocation.php

```diff
@@ -109,9 +109,11 @@ public static function deleteAccountingEntries($creditNoteId): void {
       $entityTrxn->find();
 
       while ($entityTrxn->fetch()) {
-        $trxn = new \CRM_Financial_DAO_FinancialTrxn();
-        $trxn->id = $entityTrxn->financial_trxn_id;
-        $trxn->delete();
+        if (!empty($entityTrxn->financial_trxn_id)) {
```

Comment:
why would  the `financial_trxn_id` be empty for some of the EntityFinancialTrxn records ?

Reply:
becasue the credit note has been voided, and financial trxn deleted in that case